xml/windows-ceph.xml

```diff
@@ -0,0 +1,317 @@
+
+      $diskNumber = $mappingJson.disk_number
+
+      New-VM -VMName BootFromRBD -MemoryStartupBytes 512MB
+      # The disk must be turned offline before it can be passed to &hyper; VMs
+      Set-Disk -Number $diskNumber -IsOffline $true
+      Add-VMHardDiskDrive -VMName BootFromRBD -DiskNumber $diskNumber
+      Start-VM -VMName BootFromRBD
+</screen>
+   </section>
+   <section xml:id="windows-partitions">
+     <title>&mswin; Partitions</title>
+     <para>
+       The following sample creates an empty RBD image, attaches it to the host
+       and initializes a partition:
+     </para>
+<screen>
+  rbd create blank_image --size=1G
+  rbd-nbd map blank_image
+
+  $mappingJson = rbd-nbd show blank_image --format=json
+  $mappingJson = $mappingJson | ConvertFrom-Json
+
+  $diskNumber = $mappingJson.disk_number
+
+  # The disk must be online before creating or accessing partitions.
+  Set-Disk -Number $diskNumber -IsOffline $false
+
+  # Initialize the disk, partition it and create a fileystem.
+  Get-Disk -Number $diskNumber | `
+      Initialize-Disk -PassThru | `
+      New-Partition -AssignDriveLetter -UseMaximumSize | `
+      Format-Volume -Force -Confirm:$false
+</screen>
+   </section>
+ </section>
+ <section xml:id="rbd-windows-service">
+   <title>RBD &mswin; Service</title>
+   <para>
+     In order to ensure that <command>rbd-nbd</command> mappings survive host reboots, a new
+     &mswin; service, called the &ceph; RBD Mapping Service has been created.
+     This service automatically maintains mappings as they are added using
+     the &ceph; tools. All mappings are currently persistent and being recreated
+     when the service starts, unless they are explicitly unmapped. The service
+     disconnects all mappings when being stopped.
+   </para>
+   <para>
+     This service also adjusts the &mswin; service start order so that RBD
+     images can be mapped before starting any services that may depend on it.
+     For example, VMs.
+   </para>
+   <para>
+     RBD maps are stored in the &mswin; registry at the following location:
+   </para>
+<screen>
+SYSTEM\CurrentControlSet\Services\rbd-nbd
+</screen>
+ </section>
+ <section xml:id="windows-cephfs">
+   <title>&cephfs;</title>
+   <para>
+     &ceph; for &mswin; provides &cephfs; support through the Dokany FUSE wrapper.
+     In order to use &cephfs;, install Dokany v1.3.1 or newer using the
+     installers available here: https://github.com/dokan-dev/dokany/releases
+   </para>
+   <para>
+     With Dokany installed, <filename>ceph.conf</filename> and <filename>ceph.client.admin.keyring</filename>
+     configuration files in place, &cephfs; can be mounted using the
+     <command>ceph-dokan.exe</command> command. For example:
+   </para>
+<screen>
+ceph-dokan.exe -l x
+</screen>
+   <para>
+     This command mounts the default &ceph; filesystem using the drive
+     letter <literal>x</literal>. If <filename>ceph.conf</filename> is not placed
```

Comment:
```suggestion
     letter <literal>X</literal>. If <filename>ceph.conf</filename> is not placed
```

Reply:
Windows drive letters are not case-sensitive but the UI displays them as capitals.